modules/bibauthorid/lib/bibauthorid_dbinterface.py

```diff
@@ -790,39 +790,43 @@ def get_all_paper_data_of_author(pid):
     return _select_from_aidpersonidpapers_where(select=['personid', 'bibref_table', 'bibref_value', 'bibrec', 'flag'], pid=pid)
 
 
-def get_papers_of_author(pid, claimed_only=False, include_rejected=False):  # get_all_paper_records
+def get_papers_of_author(pid, include_claimed=True, include_unclaimed=True, include_rejected=False):   ### get_all_paper_records
```

Comment:
@petros-ioannidis you are changing the signature of `claimed_only=False`, `include_claimed=True, include_unclaimed=True`, but the function is used elsewhere (e.g. in `orcidutils`). Please, upgrade everywhere...
cc @MSusik